CONTRIBUTING.rdoc

```diff
@@ -193,23 +193,23 @@ length for a comment, for example.
 
 ==== Directives
 
-For comments in markup you can add new directives (:nodoc: is a directive).
+For comments in markup you can add new directives (+:nodoc:+ is a directive).
 Directives may replace text or store it off for later use.
 
 See RDoc::Markup::PreProcess::register for details.
 
-=== JSONIndex
+=== JSON Index
 
-RDoc contains a special generator, RDoc::Generator::JSONIndex, which creates a
+RDoc contains a special generator, RDoc::Generator::JsonIndex, which creates a
 JSON-based search index and includes a search engine for use with HTML output.
 This generator can be used to add searching to any HTML output and is designed
 to be called from inside an HTML generator.
 
 == Markup
 
 Additional documentation markup formats can be added to RDoc.  A markup
-parsing class must respond to \::parse and accept a String argument containing
-the markup format.  An RDoc::Document containing documentation items
+parsing class must respond to +::parse+ and accept a String argument containing
+the markup format.  An RDoc::Markup::Document containing documentation items
```

Comment:
[note] `RDoc::Document` doesn't exist now. I probably `RDoc::Markup::Document` is correct.